src/id.rs

```diff
@@ -198,6 +198,8 @@ impl EventId {
     /// neighbors that attempt to communicate with it.
     pub const EVENT_NUM_CLOCKS_OVERFLOWED: EventId =
         EventId(unsafe { NonZeroU32::new_unchecked(EventId::MAX_INTERNAL_ID - 4) });
+    /// Raw nil value used by RaceBuffer. Must not be used as an event id elsewhere
+    pub const EVENT_NIL_VALUE_RAW: u32 = EventId::MAX_INTERNAL_ID - 5;
 
     /// The events reserved for internal use
     pub const INTERNAL_EVENTS: &'static [EventId] = &[
```

Comment:
Should your extra event be in this list?
So it shows up as an internal event in the generated event manifests, or is it intentionally excluded?
I can't remember if a user would ever see this event in their log?

Reply:
I originally had it raw to use it as a constant for a CompactLogItem trait. I just changed it to an EventId and made some functions const to be able to use it that way. The user will never see the EVENT_NIL_VALUE in the output log, but I included it in the INTERNAL_EVENTS anyway.

Reply:
Actually, as I understand, a list of internal events with descriptions is exposed to the user, so I don't think it makes sense to include it in `INTERNAL_EVENTS`